packages/instantsearch.js/src/middlewares/createInsightsMiddleware.ts

```diff
@@ -82,7 +82,7 @@ export function createInsightsMiddleware<
             window[pointer].queue.push(args);
           };
         }
-
+        window[pointer].version = VERSION;
```

Comment:
this should be in the previous branch, as otherwise it may override a different version already said?